jax/_src/debugger/core.py

```diff
@@ -84,7 +84,9 @@ def tree_flatten(self):
     flat_globals, globals_tree = _safe_flatten_dict(self.globals)
     flat_vars = flat_locals + flat_globals
     is_valid = [
-        isinstance(l, (core.Tracer, jax.Array, np.ndarray))
+        isinstance(l, (core.Tracer, jax.Array, np.ndarray)) and
+        (jax.numpy.issubdtype(l.dtype, jax.numpy.bool_) or
+         jax.numpy.issubdtype(l.dtype, jax.numpy.number))
```

Comment:
Actually, I think these are the tracers passed to the primitive bind. Should this just be `isinstance(l, core.Tracer)`? Everything else is valid as a static argument.